blueprints/data-solutions/data-platform-minimal/README.md

```diff
@@ -0,0 +1,310 @@
+# Minimal Data Platform
+
+This module implements a minimal opinionated Data Platform Architecture based on Dataproc Serverless resources. It creates and setup projects and related resources that compose an end-to-end data environment.
+
+For a complete, more versatile and configurable Data Platform, plese refer to the [Data Platform](../data-platform-foundations/) blueprint.
```

Comment:
It would be interesting to have a few paragraphs on

- what are the design differences between minimal and full
- what are the scale limits of minimal, and what use cases is this designed for or not designed for
- is there any path to upgrade from minimal to full?

Reply:
Updated README